test/dynamo/test_functionalize.py

```diff
@@ -0,0 +1,759 @@
+# Owner(s): ["module: dynamo"]
```

Comment:
nit: 1) name this file "test_auto_functionalize", 2) this should probably not be in test/dynamo ? (I'm not really sure where to put it), 3) the owner should be "module: functionalization" or "module: pt2-dispatcher"